pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -51,7 +51,7 @@ const (
 
 	// CustomResourceDefinitionSchemaVersion is semver-conformant version of CRD schema
 	// Used to determine if CRD needs to be updated in cluster
-	CustomResourceDefinitionSchemaVersion = "1.2"
+	CustomResourceDefinitionSchemaVersion = "1.3"
```

Comment:
Thanks! I've left a comment in your other PR https://github.com/cilium/cilium/pull/2772#pullrequestreview-96707777